cdc/model/sink.go

```diff
@@ -719,6 +765,112 @@ func (t *SingleTableTxn) GetCommitTs() uint64 {
 	return t.CommitTs
 }
 
+// TrySplitAndSortUpdateEvent split update events if unique key is updated
+func (t *SingleTableTxn) TrySplitAndSortUpdateEvent() error {
+	if len(t.Rows) < 2 {
+		return nil
+	}
+	newRows, split, err := trySplitAndSortUpdateEvent(t.Rows)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	// some updated events is split, need to sort
+	if split {
+		sort.Sort(txnRows(newRows))
+	}
+	t.Rows = newRows
+	return nil
+}
+
+// trySplitAndSortUpdateEvent try to split update events if unique key is updated
+// returns true if some updated events is split
+func trySplitAndSortUpdateEvent(
+	events []*RowChangedEvent,
+) ([]*RowChangedEvent, bool, error) {
+	rowChangedEvents := make([]*RowChangedEvent, 0, len(events))
+	needSplit := false
```

Comment:
`needSplit` is not a good variable name, since event is already splitter by this method.

Reply:
fixed